src/azure-cli/azure/cli/command_modules/storage/_validators.py

```diff
@@ -754,12 +754,30 @@ def validate_metadata(namespace):
 
 
 def get_permission_help_string(permission_class):
-    allowed_values = [x.lower() for x in dir(permission_class) if not x.startswith('__')]
+    allowed_values = get_permission_allowed_values(permission_class)
     return ' '.join(['({}){}'.format(x[0], x[1:]) for x in allowed_values])
 
 
+def get_permission_allowed_values(permission_class):
+    if permission_class:
+        instance = permission_class()
+
+        allowed_values = [x.lower() for x in dir(instance) if not x.startswith('_')]
```

Comment:
Is it expected to change the filter from `not x.startswith('_')` to `not x.startswith('__')`? You do want to involve sth like `_abc`, right?

Reply:
Yes, it is expected. As we don not want to introduce `_abc`, we have to use `not x.startswith('_')`. If we use `not x.startswith('__')`, `_abc` with one dash will be involved.